CounterpointGenerator/CounterpointGenerator/Note.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace CounterpointGenerator
+{
+    public class Note
+    {
+        int Note { get; set; }
```

Comment:
member names cannot be the same as their enclosing type (causes build error)

Reply:
```csharp
int Pitch { get; set; }
int Length { get; set; } = 4 // Just measure in whole beats?
```
Maybe?